modules/drivers/vertica/src/metabase/driver/vertica.clj

```diff
@@ -178,3 +197,13 @@
     (let [t (u.date/parse s)]
       (log/tracef "(.getString rs %d) [TIME_WITH_TIMEZONE] -> %s -> %s" i s t)
       t)))
+
+;; for some reason vertica `TIMESTAMP WITH TIME ZONE` columns still come back as `Type/TIMESTAMP`, which seems like a
+;; bug with the JDBC driver?
+(defmethod sql-jdbc.execute/read-column [:vertica Types/TIMESTAMP]
+  [_ _ ^ResultSet rs ^ResultSetMetaData rsmeta ^Integer i]
+  (when-let [s (.getString rs i)]
+    (let [has-timezone? (= (str/lower-case (.getColumnTypeName rsmeta i)) "timestamptz")
+          t             (u.date/parse s (when has-timezone? "UTC"))]
```

Comment:
should this be report timezone?

Reply:
good question, vertica stores column in `UTC` ([ref](https://www.vertica.com/docs/9.3.x/HTML/Content/Authoring/SQLReferenceManual/DataTypes/Date-Time/TIMESTAMP.htm))

but maybe it converts to report-tz before returning. I'll double-check.

Reply:
the value read from JDBC is in UTC even though we set `report-tz`. so parse it as `UTC` is correct.


```
(mt/with-report-timezone-id "Asia/Saigon"
 (dev/query-jdbc-db
   [:vertica 'times-mixed]
   ["select timestamp with time zone '2000-01-01 07:00:00 +7:00'"]))
;; => {:cols [{:name "?column?", :base_type :type/DateTimeWithLocalTZ}], 
;;     :rows [[#t "2000-01-01T00:00Z[UTC]"]]}

```